webpack/makeconfig.js

```diff
@@ -69,7 +69,7 @@ module.exports = function(isDevelopment) {
         loaders: isDevelopment ? [
           'react-hot', 'babel-loader'
         ] : [
-          'babel-loader'
+          'babel-loader?optional[]=optimisation.react.inlineElements'
```

Comment:
It is only dangerous if you mutate the props, which we don't do in este and is bad practice in general in terms of react. The only diff in react 0.14 is that they will freeze the props so that you get an error.